person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] You have made this function but you could have made it in less code which is shown below :+1:

```
def can_use_services?
    of_age? || @parent_permission
end
```